src/store/collections/collection-create-actions.ts

```diff
@@ -50,8 +51,18 @@ export const createCollection = (data: CollectionCreateFormDialogData) =>
             const error = getCommonResourceServiceError(e);
             if (error === CommonResourceServiceError.UNIQUE_VIOLATION) {
                 dispatch(stopSubmit(COLLECTION_CREATE_FORM_NAME, { name: 'Collection with the same name already exists.' }));
```

Comment:
This section will never be reached since `collectionService.create` call is moved outside of a try block.